pylot/control/mpc/vehicle.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+
+
+class Vehicle:
+    def __init__(self, config):
+        self.config = config
+        self.t = None  # Time [s]
+        self.s = None  # Arc distance [m]
+        self.x = None  # X coordinate [m]
+        self.y = None  # Y coordinate [m]
+        self.k = None  # Curvature [1/m]
```

Comment:
We should use more descriptive names for the state values.

Reply:
Done